tests/layer_tests/.gitignore

```diff
@@ -0,0 +1,8 @@
+#IDE FILES
+.pytest_cache
+
+#TESTS TEMP DIRECTORY
+ir/
+out/
+tf_models/
```

Comment:
why only `tf_models` are presented? It should be equal behavior for each framework